core/src/main/java/com/sequenceiq/cloudbreak/init/blueprint/DefaultBlueprintCache.java

```diff
@@ -77,7 +90,15 @@ public void loadBlueprintsFromFile() {
                                 .hostGroupCount(bp.getHostGroupCount())
                                 .description(bp.getDescription())
                                 .build();
-                        defaultBlueprints.put(bp.getName(), bpf);
+                        String fileName = split[1];
```

Comment:
maybe a validation would be nice for split size to ensure a graceful error message if it's size is not exactly 2

Reply:
already validated in blueprintUtils.isBlueprintNamePreConfigured